inventory/vagrant/group_vars/alpaca.yml

```diff
@@ -2,16 +2,14 @@
 
 # Comment in to build Alpaca from source
  alpaca_from_source: yes
- alpaca_version: dev
+ alpaca_version: one-app
```

Comment:
Can this be flipped back to 'dev' now?

Reply:
Actually it should be `2.x`. Dev is not up-to-date anymore.